base/combinatorics.jl

```diff
@@ -267,6 +267,44 @@ end
 
 invperm(P::Any32) = Tuple(invperm(collect(P)))
 
+# Row and column permutations for AbstractMatrix
```

Comment:
Should this code be moved closer to `permutecols!!` in line 119ff?

Reply:
Done!